app/ptokens/views.py

```diff
@@ -204,6 +204,12 @@ def ptoken(request, token_id='me'):
             return JsonResponse(
                 {'error': _('You must be authenticated via github to use this feature!')},
                 status=401)
+
+        if not user.has_perm('auth.user.add_pToken_auth'):
```

Comment:
@mds1 Brought up the good point that a user would be able to abuse Vue here and possibly open the modal; this ensures that they cannot make the request given their creds